src/util/sss_ldap.c

```diff
@@ -301,6 +305,14 @@ static void sss_ldap_init_sys_connect_done(struct tevent_req *subreq)
                                          "Check for certificate issues.");
                 }
 
+                if (ticks_after_install > ticks_before_install) {
+                    ret = ERR_TLS_HANDSHAKE_INTERRUPTED;
+                    DEBUG(SSSDBG_CRIT_FAILURE,
+                        "Assuming %s\n",
+                        sss_ldap_err2string(ret));
```

Comment:
Assuming <error str>? This sounds strange. Also there is bad indentation.

Reply:
It is an "addon" message - always printed after "Could not start TLS encryption."

Reply:
I've fixed the indentation. As for the message, it's a sequence of them:
```
(2021-04-13 12:45:09): [be[ldap]] [sss_ldap_init_sys_connect_done] (0x0020): ldap_install_tls failed: [Connect error] [unknown error]
(2021-04-13 12:45:09): [be[ldap]] [sss_ldap_init_sys_connect_done] (0x0020): Assuming TLS handshake was interrupted
```